src/index.html

```diff
@@ -0,0 +1,245 @@
+            </div>
+
+             <div  class="box3">
+                <span><i class="fa-regular fa-bell"></i> </span><strong>Notifications</strong>
+            </div>
+
+
+             <div  class="box4">
+                <span><i class="fa-regular fa-envelope"></i> </span>
+                <strong>Messages</strong>
+                
+            </div>
+
+
+             <div  class="box5">
+                <span><i class="fa-regular fa-bookmark"></i> </span>
+                <strong>Bookmarks</strong>
+            </div>
+
+
+             <div  class="box6">
+                <span><i class="fa-regular fa-rectangle-list"></i> </span>
+                <strong>Lists</strong>
+             </div>
+
+
+             <div  class="box7">
+                <span><i class="fa-regular fa-user"></i> </span>
+                <strong>Profile</strong>
+
+             </div>
+
+             <div  class="box8">
+                <span><i class="fa-solid fa-ellipsis"></i> </span>
+                <strong>More</strong>
+
+             </div>
+
+              <div class="tweet"> 
+                <h5>Tweet</h5>
+              </div>
+
+              <div class="footer">
+                <div class="img">
+                 <!--  <span><i class="fa-regular fa-image"></i></span></div>--> 
+                   <img src="../assets/flowers.jpeg" alt="" width="40px" height="40px" style="border-radius: 20px;"> 
+                   </div>
+
+                <div class="name"><h5>Faiza</h5></div>
+
+                <div class="dots"><span><i class="fa-solid fa-ellipsis"></i></span></div>
+              </div>
+
+            </div>
+        </div>
+           <!--  main container -->
+        <div class="main">
+            <div class="main-container1">
+                <h3>Home</h3>
+                <span><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. --><path d="M256 48a208 208 0 1 1 0 416 208 208 0 1 1 0-416zm0 464A256 256 0 1 0 256 0a256 256 0 1 0 0 512zM183.2 132.6c-1.3-2.8-4.1-4.6-7.2-4.6s-5.9 1.8-7.2 4.6l-16.6 34.7-38.1 5c-3.1 .4-5.6 2.5-6.6 5.5s-.1 6.2 2.1 8.3l27.9 26.5-7 37.8c-.6 3 .7 6.1 3.2 7.9s5.8 2 8.5 .6L176 240.5l33.8 18.3c2.7 1.5 6 1.3 8.5-.6s3.7-4.9 3.2-7.9l-7-37.8L242.4 186c2.2-2.1 3.1-5.3 2.1-8.3s-3.5-5.1-6.6-5.5l-38.1-5-16.6-34.7zm160 0c-1.3-2.8-4.1-4.6-7.2-4.6s-5.9 1.8-7.2 4.6l-16.6 34.7-38.1 5c-3.1 .4-5.6 2.5-6.6 5.5s-.1 6.2 2.1 8.3l27.9 26.5-7 37.8c-.6 3 .7 6.1 3.2 7.9s5.8 2 8.5 .6L336 240.5l33.8 18.3c2.7 1.5 6 1.3 8.5-.6s3.7-4.9 3.2-7.9l-7-37.8L402.4 186c2.2-2.1 3.1-5.3 2.1-8.3s-3.5-5.1-6.6-5.5l-38.1-5-16.6-34.7zm6.3 175.8c-28.9 6.8-60.5 10.5-93.6 10.5s-64.7-3.7-93.6-10.5c-18.7-4.4-35.9 12-25.5 28.1c24.6 38.1 68.7 63.5 119.1 63.5s94.5-25.4 119.1-63.5c10.4-16.1-6.8-32.5-25.5-28.1z"/></svg> </span>
```

Comment:
You could potentially create a separate folder for all your svgs so you don't have all these extra lines of code in your core HTML